internal/kafka/internal/environments/integration.go

```diff
@@ -45,6 +45,7 @@ func (b IntegrationEnvLoader) Defaults() map[string]string {
 		"dataplane-cluster-scaling-type":    "manual",
 		"strimzi-operator-addon-id":         "managed-kafka-qe",
 		"kas-fleetshard-addon-id":           "kas-fleetshard-operator-qe",
+		"admin-authz-config-file":           "config/admin-authz-configuration.yaml",
```

Comment:
```suggestion
		
```
not needed as it is already set here https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/fd1700778497d649ae4943d8d732c664f6361e28/pkg/auth/roles_authz.go#L38